code/modules/surgery/organs/subtypes/standard.dm

```diff
@@ -234,15 +234,15 @@
 			dna = owner.dna.Clone()
 		name = "[dna.real_name]'s head"
 		if(owner.glasses)
-			owner.unEquip(owner.glasses)
+			owner.unEquip(owner.glasses, TRUE)
 		if(owner.head)
-			owner.unEquip(owner.head)
+			owner.unEquip(owner.head, TRUE)
 		if(owner.l_ear)
-			owner.unEquip(owner.l_ear)
+			owner.unEquip(owner.l_ear, TRUE)
 		if(owner.r_ear)
-			owner.unEquip(owner.r_ear)
+			owner.unEquip(owner.r_ear, TRUE)
 		if(owner.wear_mask)
-			owner.unEquip(owner.wear_mask)
+			owner.unEquip(owner.wear_mask, TRUE)
```

Comment:
would you name the TRUE arguement? makes it slightly more readable.

Reply:
yes

Reply:
done